apps/web/src/app/page.tsx

```
@@ -26,10 +27,27 @@ const Particle = ({ delay }: { delay: number }) => (
 
 export default function AnimatedLoginPage() {
   const [showForm, setShowForm] = useState(false)
+  const [nfts, setNfts] = useState<any[]>([])
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)